src/sentry_airflow/hooks/sentry_hook.py

```diff
@@ -101,7 +104,7 @@ def __init__(self, sentry_conn_id=None):
             else:
                 self.conn_id = self.get_connection(sentry_conn_id)
             self.dsn = self.conn_id.host
-            init(dsn=self.dsn, integrations=integrations)
+            init(dsn=self.dsn, integrations=integrations, environment=os.get_env('ENVIRONMENT', ''))
```

Comment:
Sentry natively accepts `SENTRY_ENVIRONMENT` for the environment option. https://github.com/getsentry/sentry-python/blob/master/sentry_sdk/client.py#L62.

Reply:
great thanks!